src/odemis/acq/fastem.py

```diff
@@ -266,6 +279,62 @@ def __init__(self, scanner, multibeam, descanner, detector, stage, ccd, beamshif
         # Threading event, which keeps track of when image data has been received from the detector.
         self._data_received = threading.Event()
 
+    def pre_calibrate(self):
+        exception = None
+        if self._pre_calibrate and fastem_calibrations:
+            try:
+                fastem_conf.configure_scanner(self._scanner, fastem_conf.MEGAFIELD_MODE)
+                self._beamshift.shift.value = (0, 0)
```

Comment:
Does it not make sense to reset the beamshift also when you do not pre-calibrate? So that the magnetic field correction is also not running into out of range trouble?